.gitignore

```diff
@@ -52,5 +52,5 @@ test/fuzz/**/corpus
 test/fuzz/**/crashers
 test/fuzz/**/suppressions
 test/fuzz/**/*.zip
-crypto/vrf/internal/vrf/lib
-crypto/vrf/internal/vrf/include
+node/data
```

Comment:
Why is `node.js` required to build libsodium?

Reply:
There is no relation `node.js`. This is a just temporary data directory on the test.